src/apps/chifra/pkg/token/token.go

```diff
@@ -76,7 +76,7 @@ func GetState(chain string, tokenAddress base.Address, blockNumber string) (toke
 	return queryToken(chain, tokenAddress, blockNumber)
 }
 
-func GetBalanceAt(chain string, token, holder base.Address, blockNumber string) (balance *big.Int, err error) {
+func GetTokenBalanceAt(chain string, token, holder base.Address, blockNumber string) (balance *big.Int, err error) {
```

Comment:
Let's establish a convention. Should functions defined in a package have the package name in their names? Both are same to me, it's just a matter of style:
`token.GetBalanceAt` vs `token.GetTokenBalanceAt`

Reply:
I think this should be changed back to `GetBalanceAt` The object itself (token) gives enough context for what's being requested. What do you think? I don't think we should put the package name in the function name.

Reply:
As far as I know, Go stdlib also doesn't put package names in function names, so I agree